### ISSUE TITLE
Cant get hold of non warning logs

### ISSUE THREAD

**Issue by martinsson** [Updater/protractor-console#4]
Using the following config

```
    plugins: [{
        package: 'protractor-console',
        logLevels: ['debug', 'info', 'warning', 'severe']
    }],

```

I was expecting to get the normal console.log messages but I only get the console.error and console.warn messages.

Is this the expected behaviour? How could I get hold of console.log?

Thanks

**Comment by pmowrer** [comment]
What versions of `protractor-console` and `protractor` are you using?

**Comment by martinsson** [comment]
Sorry should have put it in there in the first place. The respective versions are 
protractor: 3.0.0
protractor-console: 2.0.0

**Comment by pmowrer** [comment]
This issue popped up before: https://github.com/Updater/protractor-console/issues/1

That time it was b/c of a breaking change to Protractor's plugin system in `2.2.0`. However, looking through the Protractor CHANGELOG, there doesn't appear to have been any such breaking changes in `3.0.0`.
1. Are you seeing `Test console output (log levels: warning, severe)` even when you set other log levels?
2. Would you be able to test this with Protractor `2.x` (preferably `2.2.0`)?

**Comment by martinsson** [comment]
1. Yes I see what I set. For instance
   
   Test console output (log levels: info, warning, severe)
2. I get the same behavior with protractor 2.2.0 (no console.log). I do
   however see the logs in the debugger of the opened chrome

**Comment by pmowrer** [comment]
So you've set:

``` javascript
    plugins: [{
        package: 'protractor-console',
        logLevels: ['debug', 'info', 'warning', 'severe']
    }]
```

but you're not seeing `debug` in `Test console output ...`? Does `console.info` work?

**Comment by pmowrer** [comment]
I see you said "I see what I set" so that last comment may be redundant (but just to be sure).

If you're actually seeing what you set, it means the options are being applied correctly. Not sure what the problem might be then. Total shot in the dark, but does your terminal use a color scheme that may mask the output of `debug` (and `info`?) level statements?

**Comment by pmowrer** [comment]
I'm going to guess it has to do with the browser/driver's logging setting. For example, it appears the Chrome driver throws out anything below WARN unless configured otherwise. 

https://github.com/angular/protractor/issues/2207
https://github.com/webdriverio/webdriverio/issues/491
https://github.com/angular/protractor/blob/master/docs/faq.md#how-can-i-get-hold-of-the-browsers-console
https://github.com/SeleniumHQ/selenium/wiki/DesiredCapabilities#loggingpreferences-json-object

Playing around with the `loggingPrefs` capabilities setting, I still wasn't able to have Chrome output anything but WARNING and SEVERE. Perhaps play around with it and see what you find?

**Comment by martinsson** [comment]
So you get the same behavior with chrome?

Will play around with it indeed.

**Comment by pmowrer** [comment]
Yes, only WARNING and SEVERE in Chrome as far as I can tell. That appears to be the default (per links above).

**Comment by pmowrer** [comment]
Where you able to make it work @martinsson?

**Comment by martinsson** [comment]
Sorry for the delay, and thanks for reminding me, I got busy elsewhere. I just tried and found the following snippet to work 

```
capabilities: {
    browserName: 'chrome', 
    loggingPrefs: {
        'driver': 'INFO',
        'browser': 'INFO'
    }
},
```

basically the answer you pointed out [webdriverio/webdriverio#491](https://github.com/webdriverio/webdriverio/issues/491)
with renamed parameters

**Comment by pmowrer** [comment]
Thanks for the update! I'll try it out and put instructions in the repo README when I get a chance.

**Comment by sakovias** [comment]
@martinsson Thank you! Your snippet is such a life saver!
